### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by rov9**
E.commerce Home Page

### CODE REVIEW

**Comment by rov9** [comment]
Netlify link:https://naughty-johnson-fdb731.netlify.app/

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <li class="link-item"><a href="#" class="link">Home</a></li>
+        <li class="link-item"><a href="#" class="link">Product page</a></li>
+        <li class="link-item"><a href="#" class="link">Wishlist</a></li>
+        <li class="link-item"><a href="#" class="link">My Cart</a></li>
+    </ul>
+    </nav>
+
+    <header class="hero-section">
+        <div class="content">
+            <!-- <img src="/logo.png" class="logo" alt=""> -->
```
Comment (iamsaurav28) [inline index.html:38]: just a suggestion u can delete commit parts. overall lokks good

**Review by iamsaurav28 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (rov9) [reply index.html:38]: Thanks a lot :)

File: style.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    width: 80%;
+    height: 40px;
+    padding: 10px;
+    border-top-left-radius: 10px;
+    border-bottom-left-radius: 10px;
+    border: 1px solid var(--border-color);
+    text-transform: capitalize;
+    background: none;
+    color: #a9a9a9;
+    outline: none;
```
Comment (ImrozNezami) [inline style.css:46]: Instead of outline:none use 
 outline: 2px solid transparent;
 outline-offset: 2px;

File: style.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+
+.search-btn{
+    width: 20%;
+    height: 40px;
+    padding: 10px 20px;
+    border: none;
+    outline: none;
+    cursor: pointer;
+    background: #383838;
```
Comment (ImrozNezami) [inline style.css:57]: use css variable for color.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+.hero-section{
+    width: 100%;
+    height: calc(100vh - 120px);
+    background-image: url('/background-image.png');
+    background-size: cover;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
```
Comment (ImrozNezami) [inline style.css:101]: use prettier extension, it will remove extra space and keep the code neat and clean.

**Pull request merged**